NEWS.md

```diff
@@ -1,3 +1,9 @@
+# moderndive 0.5.3.9001
```

Comment:
Make the next entry in the NEWS 0.5.4.

Reply:
Addressed in commit [ca6909b](https://github.com/beanumber/moderndive/pull/13/commits/ca6909b9736eba57348998d5faff57cd6202f109)